ui/app/components/clients/activity.ts

```diff
@@ -92,10 +92,11 @@ export default class ClientsActivityComponent extends Component<Args> {
     const { namespace, activity, mountPath } = this.args;
     // only do this if we have a mountPath filter.
     // namespace is filtered on API layer
-    if (activity?.byNamespace && namespace && mountPath) {
-      return filteredTotalForMount(activity.byNamespace, namespace, mountPath);
+    if (activity?.byMonth && namespace && mountPath) {
+      return filteredTotalForMount(activity.byMonth, namespace, mountPath);
     }
-    return activity?.total;
+
+    return activity.newClientTotal;
```

Comment:
Per comment below, I believe this should be reverted because these usage counts should still map to the top-level `total` block for the date range.